extension/ui/components/conversation/AgentMessage.tsx

```diff
@@ -607,6 +608,67 @@ export function AgentMessage({
       );
     }
 
+    const generatedFiles = agentMessage.actions
+      .flatMap((action) => action.generatedFiles ?? [])
+      .filter((file) => !file.hidden);
+
+    const downloadGeneratedFile = async (file: ActionGeneratedFileType) => {
+      try {
+        const token = await platform.auth.getAccessToken();
+
+        const response = await fetch(
```

Comment:
There's no method in the sdk for this? Should we add one instead of doing it raw?

Reply:
No method for this.
So far all fetch calls in the extension are done "raw".
I can add one 👍🏾

Reply:
Handled in https://github.com/dust-tt/dust/pull/17194 as an other function `downloadFile` has been created since